StripeCore/StripeCore/Source/Analytics/AnalyticLoggableError.swift

```diff
@@ -139,4 +141,13 @@ extension AnalyticLoggableError where Self: Error {}
             return String((error as NSError).code)
         }
     }
+
+    static func extractStripeAPIRequestID(from error: Error) -> String? {
+        let error = error as NSError
```

Comment:
Is there ever a time we would want to log the request ID on something that is not an error?

Reply:
I can't immediately think of a reason, but possibly!  I feel like I'm only looking at request IDs to investigate problems.